parm/config/config.resources.emc.dyn

```diff
@@ -411,7 +411,7 @@ elif [ $step = "postsnd" ]; then
         export npe_postsnd=13
         export npe_node_postsnd=4
     fi
-    if [[ "$machine" = "HERA" ]]; then export npe_node_postsnd=2; fi
+    if [[ "$(echo "$npe_node_postsnd * $nth_postsnd" | bc)" -gt "$npe_node_max" ]]; then export npe_node_postsnd=$(echo "$npe_node_max / $nth_postsnd" | bc); fi
```

Comment:
This is too long. Break it into a regular indented block.

Reply:
Done, thanks!